Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -3000,6 +3004,9 @@ class CloneDeserializer : CloneBase {
         case CryptoAlgorithmIdentifierTag::ECDH:
             result = CryptoAlgorithmIdentifier::ECDH;
             break;
+        case CryptoAlgorithmIdentifierTag::Ed25519:
+            result  = CryptoAlgorithmIdentifier::Ed25519;
```

Comment:
As I said in a past review, it would be good to have a test for key serialisation.
That could be covered by using postMessage with a Ed25519 key.

Reply:
I added a tests searializationt that imports and exports the key, is that what you meant ? Maybe I did get you wrong